r_basics_practice/r_basics_practice.md

```diff
@@ -0,0 +1,149 @@
+
+If you find yourself feeling overwhelmed or frustrated, take a step back. 
+Sometimes ten minutes away from the keyboard is all you need to come at a problem with a fresh mindset.
+You can also try going back to review earlier examples in previous modules you've completed to refresh yourself on how those commands work in a more scaffolded context.
+
+And remember that [learning to ask for help with code](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/learning_to_learn/learning_to_learn.md#8) is also a important skill, and one you should be practicing! 
+If you get stuck, ask for help!
+
+</div>
+
+## The Data
+
+These data are from a COVID-19 serological survey conducted in Yaounde, Cameroon (Nwosu, K., Fokam, J., Wanda, F. et al., 2021[^1](Kene David Nwosu, Joseph Fokam, Franck Wanda, Lucien Mama, Erol Orel, Nicolas Ray, Jeanine Meke, Armel Tassegning, Desire Takou, Eric Mimbe, Beat Stoll, Josselin Guillebert, Eric Comte, Olivia Keiser, & Laura Ciaffi. 2021. kendavidn/yaounde\_serocovpop\_shared: Initial release v1.0.0. Zenodo. https://doi.org/10.5281/zenodo.5218965)). The authors have made all of the code and data publicly available under a [creative commons 4.0 license](https://creativecommons.org/licenses/by/4.0/legalcode) to facilitate re-use.
+
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+To learn more about the study, see the [zenodo page for this dataset](https://zenodo.org/record/5218965#.YeBq2RPMITW). You can also read the published article online: [SARS-CoV-2 antibody seroprevalence and associated risk factors in an urban district in Cameroon](https://www.nature.com/articles/s41467-021-25946-0).
+
+</div>
+
+## Lesson Preparation
+
+@lesson_prep_r
+
+## Practice Problem
+
+Time to jump in! 
+Open the `basics_practice_exercises.Rmd` file in RStudio. 
+
+This file includes a list of instructions and questions for you to answer.
+You'll need to add code chunks to the .Rmd file, and write R code to complete the tasks. 
+Good luck, and have fun!
+
+<div class = "help">
+<b style="color: rgb(var(--color-highlight));">Troubleshooting help</b><br>
+
+If you're having trouble, remember that the skills you'll need in this practice module are (almost!) all covered in previous modules in the R Basics sequence. 
+
+You can go back to review those modules, or even just look over the exercise and solution files for them, which are also available in the same `education_r_environment` directory you're already working in. 
+In particular, the files for `r_basics_introduction`, `r_basics_visualize_data`, and `r_basics_transform_data` will be helpful.
+
+</div>
+
+### Stuck?
+
+If this practice problem was challenging, [here is a link to a solutions file](https://github.com/arcus/education_r_environment/blob/main/r_basics_practice/basics_practice_solutions.Rmd) where we go through example code for the practice problems.
```

Comment:
It might be worth pointing out that if they're using Posit Cloud, the solutions are already in the environment... though this may be obvious.

Reply:
Good call! I'll put something in :)